src/output/icons.rs

```diff
@@ -198,16 +212,20 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     ".zsh_sessions"       => Icons::SHELL,          // 󱆃
     "._DS_Store"          => Icons::OS_APPLE,       // 
     "a.out"               => Icons::SHELL_CMD,      // 
-    "authorized_keys"     => '\u{f08c0}',           // 󰣀
```

Comment:
This is a SSH specific and should not be changed to a license icon.

Reply:
I just thought keys -> keys icon